seed/data_importer/views.py

```diff
@@ -236,6 +240,178 @@ def create(self, request):
+                    # However, we need to be sure to not add the flagged bad strings.
+                    # However, a flagged value *could* be a value property name, and we would want to allow that
+                    if isinstance(this_pm_variable, basestring):
+                        if pm_variable == u'property_name':
+                            this_row.append(this_pm_variable)
+                            added = True
+                        elif this_pm_variable not in pm_flagged_bad_string_values:
+                            this_row.append(this_pm_variable)
+                            added = True
+
+                    # If it isn't a string, it should be a dictionary, storing numeric data and units, etc.
+                    else:
+
+                        # As long as it is a valid dictionary, try to get a meaningful value out of it
+                        if '#text' in this_pm_variable and this_pm_variable['#text'] != 'Not Available':
+
+                            # Coerce the value into a proper set of Pint units for us
+                            new_var = LocalUploaderViewSet._get_pint_var_from_pm_value_object(this_pm_variable)
+                            if new_var['success']:
+                                pint_value = new_var['pint_value']
+                                this_row.append(pint_value.magnitude)
+                                added = True
+                                # TODO: What to do with the pint_value.units here?
+
+                # And finally, if we haven't set the added flag, give the csv column a blank value
+                if not added:
+                    this_row.append(u'')
+
+            # Then add this property row of data
+            rows.append(this_row)
+
+        # Then write the actual data out as csv
+        with open(path, 'wb') as csv_file:
+            pm_csv_writer = csv.writer(csv_file)
+            for row in rows:
+                pm_csv_writer.writerow(row)
+
+        # Look up the import record (data set)
+        import_record_pk = request.data['import_record_id']
+        try:
+            record = ImportRecord.objects.get(pk=import_record_pk)
+        except ImportRecord.DoesNotExist:
+            # clean up the uploaded file
+            os.unlink(path)
+            return JsonResponse({
+                'success': False,
+                'message': "Import Record %s not found" % import_record_pk
+            })
+
+        # Create a new import file object in the database
+        f = ImportFile.objects.create(import_record=record,
+                                      uploaded_filename='PortfolioManagerImport',
+                                      file=path,
+                                      source_type=SEED_DATA_SOURCES[PORTFOLIO_RAW],
+                                      **{'source_program': 'PortfolioManager', 'source_program_version': '1.0'})
```

Comment:
Is this the right source type/program/version for this import?  There isn't any info in the PM export to specify a version or anything like that.